test/ci/buildkite/run_pipeline.sh

```diff
@@ -22,6 +22,10 @@ elif [ "${BUILDKITE_PIPELINE_NAME}" == "cpu-intel-tf" ]; then
    export TF_LOCATION=/localdisk/buildkite-agent/prebuilt_intel_tensorflow
    export NGRAPH_TF_BACKEND=CPU
    PIPELINE_STEPS+=" ${SCRIPT_DIR}/cpu.yml "
+elif [ "${BUILDKITE_PIPELINE_NAME}" == "igpu" ]; then
+   export NGRAPH_TF_BACKEND=GPU
+   export TF_LOCATION=/localdisk/buildkite-agent/prebuilt_tensorflow_2_2_0
```

Comment:
If `TF_LOCATION` is the same for all pipelines, can you please hoist it out of the conditional?

Reply:
It differs in case of Intel-TF.

Reply:
Hoisted all but intel-tf

Reply:
Then it should only be set for the Intel TF case.